src/main/kotlin/no/nav/syfo/oppfolgingstilfelle/OppfolgingstilfelleService.kt

```diff
@@ -36,4 +46,9 @@ class OppfolgingstilfelleService(
         val dodsdato = connection.getDodsdato(personIdent)
         oppfolgingstilfellePerson?.toOppfolgingstilfellePerson(dodsdato)
     }
+
+    private fun List<Oppfolgingstilfelle>.sortedByTilfelleStartFutureExcluded(): List<Oppfolgingstilfelle> =
+        this.filter {
+            it.start.isBefore(tomorrow())
+        }.sortedByDescending { tilfelle -> tilfelle.start }
```

Comment:
Kunne ikke dette bare vært: 
```suggestion
        }.sortedByDescending { it.start }
```
Eller er det noen spesifikk grunn til å gi den et navn her og ikke i filteret?

Reply:
Jepp, kan endre til `it`.